Sources/RInAppMessaging/Models/Requests/GetConfigRequest.swift

```diff
@@ -24,6 +26,9 @@ extension GetConfigRequest {
         queryItems.append(URLQueryItem(name: CodingKeys.appVersion.rawValue, value: appVersion))
         queryItems.append(URLQueryItem(name: CodingKeys.sdkVersion.rawValue, value: sdkVersion))
         queryItems.append(URLQueryItem(name: CodingKeys.locale.rawValue, value: locale))
+        if let rmcSdkVersion = rmcSdkVersion {
```

Comment:
@SoumenRautray Normally we only handle Xcode >= 14.1 and Swift >= 5.7, so I think that you can do:
```
if let rmcSdkVersion {
```
Please confirm.
cc @imaginaris

Reply:
@SoumenRautray This change is part of Swift 5.7:
https://blog.swiftify.com/whats-new-in-swift-5-7-e0a2ba39fb78

So you can do `if let rmcSdkVersion {`, thanks.

Reply:
Ah true. We can start using this syntax improvement in all projects now 👍